tools/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]

```

Comment:
I'd probably prefer we resolve the issues to stay inside PEP8 guidelines. I can work on this separately/later, if so desired, though

Reply:
The two ignores are for line too long, which I feel is overly restrictive, and under indent, which exacerbates the line too long. This especially is a problem when full word function names are used for clarity, putting line breaks and parentheses all over the place. If you are able to reduce to 79 line length while maintaining readability though, go ahead, I'm not opposed to it. I just don't know how to get there without making an unreadable mess.